README.md

```diff
@@ -167,13 +169,13 @@ brew install $(curl -Ls -o /dev/null -w %{url_effective} https://aka.ms/InstallA
 You can install the edge build on Ubuntu Xenial with the following command:
 
 ```bash
-curl -Ls -o azure-cli_xenial_all.deb https://aka.ms/InstallAzureCliXenialEdge && dpkg -i azure-cli_xenial_all.deb
+curl --location --silent --output azure-cli_jammy.deb https://aka.ms/InstallAzureCliJammyEdge && dpkg -i azure-cli_jammy.deb
```

Comment:
Should we distinguish between AMD64 and ARM64?

Reply:
x86 architecture is still the mainstream, and AMD64 version should be enough for edge test.